app/templates/components/add-env-var.hbs

```diff
@@ -23,7 +25,28 @@
       showValidationStatusIcons=false
       as |field|
     }}
-      {{field.input  class='env-value' placeholder='Value'}}
+      {{field.input class='env-value' placeholder='Value'}}
+    {{/form.field}}
+  </div>
+  <div class="form-elem">
+    {{#form.field
+      label='Branch'
+      value=branch
+      required=false
+      class='env-branch-selector'
+      showValidationStatusIcons=false
+      as |field|
+    }}
+      {{#field.select
+        searchField='name'
```

Comment:
If the results will be mapped by `name`, then `searchField` should be removed (as it will be plain array of strings)
```suggestion
```